src/Tribe/Views/V2/View.php

```diff
@@ -1001,7 +1019,8 @@ protected function setup_template_vars() {
 			'date_formats'      => (object) [
 				'compact'        => Dates::datepicker_formats( tribe_get_option( 'datepickerFormat' ) ),
 				'month_and_year' => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
-			]
+			],
+			'messages'           => $this->get_messages(),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 10 space(s) between "'messages'" and double arrow, but found 11.

<sub>hash: fee3d830f0691792a6dc61051d7eb955</sub>